server/trillian_log_signer/main.go

```diff
@@ -39,6 +42,9 @@ var (
 	numSeqFlag               = flag.Int("num_sequencers", 10, "Number of sequencer workers to run in parallel")
 	sequencerGuardWindowFlag = flag.Duration("sequencer_guard_window", 0, "If set, the time elapsed before submitted leaves are eligible for sequencing")
 	dumpMetricsInterval      = flag.Duration("dump_metrics_interval", 0, "If greater than 0, how often to dump metrics to the logs.")
+	forceMaster              = flag.Bool("force_master", false, "If true, assume master for all logs")
```

Comment:
Yeah, but there could well be situations where it's very helpful to be able to spin up a signer that doesn't depend on etcd to get on with the job...